CMakeLists.txt

```diff
@@ -317,20 +317,17 @@ else()
     add_definitions(-DUPNP_HAS_EXTRA_HEADERS_LIST)
 endif()
 
-find_package(fmt REQUIRED)
-target_link_libraries(gerbera fmt::fmt)
 
 # See if it's just installed as a library first, as not all installs have .cmake files
 find_library(SPDLOG_LIBRARY spdlog)
 if(SPDLOG_LIBRARY)
     set(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} ${SPDLOG_LIBRARY})
-    target_link_libraries(gerbera ${SPDLOG_LIBRARY})
+    target_link_libraries(libgerbera ${SPDLOG_LIBRARY})
 else()
     # Didn't find it, so try the .cmake file, fail if not found
     find_package(spdlog REQUIRED)
-    target_link_libraries(gerbera spdlog::spdlog)
+    target_link_libraries(libgerbera spdlog::spdlog)
```

Comment:
spdlog needs to be linked to `libgerbera` because it includes its headers. It compiles if you dump all headers into a single system path, but if it's a really separate path, Gerbera won't build.